lightspeed/modules/proc_resolve-ansible-lint-rule-violations.adoc 

```diff
@@ -0,0 +1,53 @@
+For more information about autofixes, see link:https://ansible.readthedocs.io/projects/lint/autofix/[Autofix] in Ansible Lint Documentation. 
+
+* *Using syntax-checks*
++
+Ansible-lint also performs syntax checks while scanning your Ansible files. If any syntax-check errors are found, ansible-lint stops processing the files. 
+For more information about syntax-check errors, link:https://ansible.readthedocs.io/projects/lint/rules/syntax-check/[syntax-check] in Ansible Lint Documentation.
++
+The {AnsibleContentParser} handles syntax-check rule violations in the following manner:
+
+** If `syntax-check` errors are found in the first execution of ansible-lint, {AnsibleContentParser} generates a list of files that contain the rule violations.
+
+** If one or more `syntax-check` errors are found in the first execution of ansible-lint, the {AnsibleContentParser} runs ansible-lint again but excludes the files with syntax-check errors. After the scan is completed, the {AnsibleContentParser} generates a list of files that contain rule violations. The list includes all files that caused syntax-check errors as well as other rule violations.  The {AnsibleContentParser} excludes files with rule violations in all future scans, and the final training data set does not include data from the excluded files.
+
+.Procedure
+Use one of the following methods to resolve ansible-lint rule violations:
+
+* Run the {AnsibleContentParser} with the  `--no-exclude` option
+If any rule violations, including syntax-check errors, are found, the execution is aborted with an error and no training data set is created.
+
+* Limit the set of rules that ansible-lint uses to scan your data with the  `--profile` option
++
+It is recommended that you fix the files with rule violations. However, if you do not want to modify the source files, you can limit the set of rules that ansible-lint uses to scan your data. 
+To limit the set of rules that ansible-lint uses to scan your data, specify the --profile option with a predefined profile (for example, minimum, basic, moderate, safety, shared, or production profiles) or by using ansible-lint configuration files. For more information, see the link:https://ansible.readthedocs.io/projects/lint/[Ansible Lint documentation]. 
+
+* Run the {AnsibleContentParser} by skipping ansible-lint preprocessing
+You can run the content parser without ansible-lint preprocessing. The {AnsibleContentParser} generates a training data set without scanning for ansible-lint rule violations. 
++
+To run {AnsibleContentParser} without ansible-lint preprocessing, execute the following command: +
+`$ ansible-content-parser _source_ _output_ _-S_`
++
+Where:
++
+*** `_source_`: Specifies the source of the training data set.
+*** `_output_`: Specifies the source of the training data set.
```

Comment:
Should this read as "Specifies the output of the training data set" instead of "the source"?

Reply:
Yes!! thank you for catching this - I've updated the instances.